dockers/docker-orchagent/Dockerfile.j2

```diff
@@ -24,9 +24,11 @@ RUN apt-get update &&       \
         python3-dev
 
 {% if ( CONFIGURED_ARCH == "armhf" or CONFIGURED_ARCH == "arm64" ) %}
-# Fix for gcc/python not found in arm docker
+# Fix for gcc/python/iputils-ping not found in arm docker
 RUN apt-get install -f -y python2.7 python2.7-dev
-RUN apt-get install -y gcc-8
+RUN apt-get install -y   \
+        gcc-8            \
```

Comment:
why do we install gcc-8 here?

Reply:
I am not sure about that. The gcc package were introduced two years ago: https://github.com/Azure/sonic-buildimage/pull/3419

And were updated last year: https://github.com/Azure/sonic-buildimage/pull/5749/files

@antony-rheneus do you have any insight?

Reply:
sorry, my read is bad. you are good here.